src/testUtils/index.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import {promises as fs} from 'fs';
+
+export function removeFileExtension(fileName) {
+  return fileName.substr(0, fileName.lastIndexOf('.'));
+}
+
+function removeUnknown(e) {
+  return e !== 'Unknown';
+}
+
+export async function getStatesAndDistrictsFromAPI() {
+  const stateDistrictWiseResponse = (
+    await axios.get('https://api.covid19india.org/state_district_wise.json')
+  ).data;
+  const states = Object.keys(stateDistrictWiseResponse).filter(removeUnknown);
+  const result = {};
+  states.map((stateName) => {
+    result[stateName] = Object.keys(
+      stateDistrictWiseResponse[stateName]['districtData']
+    ).filter(removeUnknown);
+  });
+  return result;
+}
+
+export async function getStatesAndDistrictsFromMaps() {
```

Comment:
A snippet of the output. The function traverses through the `public/maps` directory and gets the collection. We receive an object with keys set to the name of the state and the districts of the state as values.
```
{
        Puducherry: [ 'Puducherry', 'Karaikal', 'Mahe', 'Yanam' ],
        'Andhra Pradesh': [
          'Srikakulam',    'Vizianagaram',
          'Visakhapatnam', 'Krishna',
          'Guntur',        'East Godavari',
          'Prakasam',      'Kurnool',
          'Anantapur',     'S.P.S. Nellore',
          'West Godavari', 'Y.S.R.',
          'Chittoor'
        ],
}
```